README.md

```diff
@@ -1 +1,38 @@
-# docker-selenium-screenshot
\ No newline at end of file
+# docker-selenium-screenshot
+
+Container for taking screenshots using selenium
+
+
+## Allowed arguments
+1. **url**: the URL of the page to take the screenshot
+2. **file_name**: Name of the image which will be saved (e.g `/path/of/my/image.png`)
+3. **width**: Initial width for the browser
+4. **height**: Initial height for the browser
+5. **app_selector_xpath** (*optional*): XPath selector for search the target content in the DOM. Default `//div[@id='root']`
+6. **not_load_selector_xpath** (*optional*): XPath selector when it can't find the content. Default `div[contains(@class, 'selenium-data-not-loaded')]`
+
+**Note**: Everything arguments are positional, each one must be sent in the order described above.
```

Comment:
There's a typo here, it should say: "Every argument is positional" or "Each one of the arguments are positional".